style.css

```diff
@@ -1,446 +1,440 @@
+  border-top: 2px solid var(--dark-grey);
+  display: grid;
+  padding: 12px 16px 12px 16px;
+  grid-template-columns: 65px auto;
 }
 
 .tweet-head {
-    height: 55px;
-    display: grid;
-    align-items: center;
+  display: grid;
+  align-items: center;
+  line-height: 20px;
+  margin-bottom: 10px;
 }
 
 .align_right {
-    grid-template-rows: 55px auto 50px;
+  grid-template-rows: auto auto 50px;
 }
 
 .content_image {
-    max-width: 500px;   
-    border-radius: 20px;
-    justify-self: right;
+  max-width: 500px;
+  border-radius: 20px;
+  justify-self: right;
+  max-height: 650px;
 }
 
-
 .card .list_items_in_row {
-    justify-content: space-between;
-    padding-right: 100px;
-    margin-top: 2px;
-    height: 32px;
+  justify-content: space-between;
+  padding-right: 100px;
+  margin-top: 2px;
+  height: 32px;
+}
 
+.hover_blue:hover,
+.hover_green:hover,
+.hover_red:hover {
+  border-radius: 50px;
+  padding: 5px;
 }
 /* MAIN - CONTENT - ends here */
 
 /* RIGHT PANEL starts here */
-
 .right-panel {
-    padding: 5px 0px 0px 30px;
-    max-height: 100vh;
+  padding: 5px 0px 0px 25px;
+  max-height: 100vh;
 }
+
 .scroll {
-    max-height:  calc(100vh - 68px);
-    overflow-y: scroll;
+  max-height: calc(100vh - 68px);
+  overflow-y: scroll;
 }
 
-
 .round {
-    background-color: var(--dark-grey);
-    border-radius: 25px;
-    margin-bottom: 15px;
+  background-color: var(--dark-grey);
+  border-radius: 20px;
+  margin-bottom: 15px;
 }
 
-.round * {
-    background-color: var(--dark-grey)
+.round_bottom {
+  border-radius: 0px 0px 25px 25px;
 }
 
-.search-bar {
-    display: grid;
-    grid-auto-flow: column;
-    justify-content: left;
-    align-items: center;
-    padding: 10px;
-    padding-left: 20px;
+.round > * {
+  background-color: var(--dark-grey);
+}
 
+.search-bar {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: left;
+  align-items: center;
+  padding: 10px;
+  padding-left: 20px;
 }
 
 .search-bar * {
-    color: var(--grey-);
+  color: var(--grey-);
 }
 
-
 .right-panel .search-bar p {
-    margin-left: 10px;
-}
-
-.trends-head{
-    display: grid;
-    grid-template-columns: 1fr auto;
-    border-radius: 25px 25px 0px 0px;
-}
-
-.round-top-corners {
-    border-radius: 25px 25px 0px 0px;
+  margin-left: 10px;
 }
 
-.round-bottom-corners {
-    border-radius: 0px 0px 25px 25px;
+.right_panel_heads {
+  display: grid;
+  grid-template-columns: 1fr auto;
 }
 
 .trends_item {
-    padding: 12px 16px 12px 16px;
-    line-height: 20px;
+  line-height: 20px;
 }
 
+.two_col_1fr_auto {
+  display: grid;
+  grid-template-columns: 1fr auto;
+}
 
 .follow {
-    padding: 15px 15px 15px 20px;
-    line-height: 25px;
+  line-height: 25px;
 }
 
-
-.follow_button {
-    background-color: var(--light-grey);
-    color: black;
-    font-weight: bold;
-    width: 75px;
-    height: 30px;
-    border-radius: 50px;
-    display: grid;
-    place-items: center;
+.right_items_padding {
+  padding: 12px 16px 12px 16px;
```

Comment:
You could use `padding: 12px 16px`

Reply:
Yes that is true.  :) i did not even notice.